module/reports/helper.js

```diff
@@ -101,63 +102,7 @@ module.exports = class ReportsHelper {
 				projectReport[CONSTANTS.common.SUBMITTED_STATUS] = 0
 				projectReport[CONSTANTS.common.INPROGRESS_STATUS] = 0
 				projectReport[CONSTANTS.common.STARTED] = 0
-
-				let types = await this.types()
```

Comment:
@MallanagoudaB why these codes are removed?

Reply:
Because its not using anywhere below